backend/src/Response/Admin/Order/OrderByIdGetForAdminResponse.php

```diff
@@ -101,4 +101,6 @@ class OrderByIdGetForAdminResponse
     public $phone;
 
     public int|null $paidToProvider;
+
+    public string|null $storeOwnerName;
```

Comment:
just as a note, as long as their will always be a name for the store, null can be avoided.